src/receive_imf.rs

```diff
@@ -2211,7 +2211,7 @@ async fn check_verified_properties(
         let peerstate = Peerstate::from_addr(context, &to_addr).await?;
 
         // mark gossiped keys (if any) as verified
-        if mimeparser.gossiped_addr.contains(&to_addr) {
+        if mimeparser.gossiped_addr.contains(&to_addr.to_lowercase()) {
```

Comment:
ftr: there is another `gossiped_addr.contains` at line 249, that, however, compares against `recipient.addr` which is lowercased in `get_all_addresses_from_header()`, so things are fine.

not sure, however, if that is covered by a test as well. maybe add another `to_lowercase` there as well, this also protects a bit when the pattern is copied at some point in the future :)